contracts/test/RewardsStub.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity 0.8.6;
+
+import "../../contracts/Rewards.sol";
+
+contract RewardsStub is Rewards {
+    uint32[] internal operators;
+    mapping(uint32 => uint256) internal operatorWeight;
+    mapping(uint32 => uint256) internal withdrawnRewards;
+
+    function addOperator(uint32 operator, uint256 weight) public {
+        operators.push(operator);
+        operatorWeight[operator] = weight;
+        operatorRewards[operator].accumulated = globalRewardAccumulator;
+        updateOperatorRewards(operator, uint32(weight));
+    }
+
+    function syncRewards(uint32 operator) public {
```

Comment:
It looks that this function is called only from `withdrawRewards`. Why don't we call `updateOperatorRewards` directly from there? This is what we are doing in `SortitionPool.withdrawRewards`.

Reply:
Addressed :+1: